python/smqtk/data_rep/data_element_impl/url_element.py

```diff
@@ -61,3 +61,6 @@ def get_bytes(self):
         else:
             raise RuntimeError("Request response not OK. Status code returned: "
                                "%d", r.status_code)
+
+
+#DATA_ELEMENT_CLASS = DataUrlElement
```

Comment:
Originally DATA_ELEMENT_CLASS = DataUrlElement was not in that file at all. I had just put it in there to test that implementation. I'm not sure why it wasn't there to begin with, but didn't want to add it without being sure why it wasn't there.